cluster/restore-from-backup.sh

```diff
@@ -101,7 +104,7 @@ wait_for_cluster_healthy() {
 # Wait until etcd and apiserver pods are down.
 wait_for_etcd_and_apiserver_down() {
   for i in $(seq 120); do
-    etcd=$(docker ps | grep etcd | grep -v etcd-empty-dir | grep -v etcd-monitor | wc -l)
+    etcd=$(docker ps | grep -e etcd-server-gke -e etcd-server-events-gke | wc -l)
```

Comment:
This looks like it's making the script gke specific. If it only works on GKE, why have it in the k/k repository?

Reply:
Yeah. Initially this script was supposed to be generic, but over time it appeared some GKE/GCE-specific things may be needed. But when I'm looking into it now, maybe that's the only one.

Maybe add a TODO to make it work for all platforms.

Reply:
The reason for the change is that it doesn't work in GKE right now: there is a plenty of containers with etcd in name (not only etcd-empty-dir, etcd-monitor) and this waits forever.

Will try to solve this issue in some other way.